docs/javascript/publish-nodejs-app-azure.md

```diff
@@ -146,7 +146,7 @@ how to start the Node.js process. Details of the implementation can be seen at
 * You can connect to the running container via SSH for investigations. This is easily done using the Azure Portal. Select the App Service, and scroll down
 the list of tools until reaching **SSH** under the **Development Tools** section.
 * To aid in troubleshooting, go to the **Diagnostics logs** settings for the App Service, and change the **Docker Container logging** setting
-from **Off** to **File System**. Logs are created in the container under */home/LogFiles/*_docker.log*, and can be accessed on the box using SSH or FTP(S).
+from **Off** to **File System**. Logs are created in the container under */home/LogFiles/*_docker.log*, and can be accessed on the toolbox using SSH or FTP(S).
```

Comment:
I don't think this is correct. I think box might refer to "computer" or "machine".

Reply:
Thanks @gewarren , if this is not "toolbox", I will need a confirmation so the linguist can translate it accordingly. 
Please, let me know if we can use computer instead. 
@Mikejo5000